src/workerd/api/node/crypto.h

```diff
@@ -385,6 +385,26 @@ class CryptoImpl final: public jsg::Object {
       jsg::BufferSource buffer,
       PublicPrivateCipherOptions options);
 
+  struct CipherInfo {
+    kj::String name;
+    int nid;
+    jsg::Optional<int> blockSize;
+    jsg::Optional<int> ivLength;
+    int keyLength;
+    kj::String mode;  // 'cbc', 'ccm', 'cfb', 'ctr', 'ecb', 'gcm', 'ocb',
```

Comment:
Maybe a todo to make this an enum to avoid serialization?

Reply:
eh, this isn't that much of a hot path method in any use I've ever seen it in. Not convinced it is worth the added complexity of an enum.